### PR TITLE
Data model for the emline files

### PR DESCRIPTION
**Pull request by araichoor**
This PR updates the two datamodel `*.rst` files for the emline files.
The description is done in `emline-SPECTROGRAPH-TILEID-GROUPID.rst`, to which refers `emline-SURVEY-PROGRAM-PIXNUM.rst`.

I know the test will fail with being unhappy with the `1e-17 erg/s/cm^2`-like units; though I don't know what is the correct approach here, so I'd appreciate guidelines.

Thanks!

### CODE REVIEW

File: doc/DESI_SPECTRO_REDUX/SPECPROD/tiles/GROUPTYPE/TILEID/GROUPID/emline-SPECTROGRAPH-TILEID-GROUPID.rst [hunk 1]
```diff
@@ -53,113 +47,127 @@ Required Header Keywords
...
+================= ======= =============================== ===================
+TARGETID          int64                                   Unique targeting ID
+Z                 float64                                 Redshift used for fitting (from the redrock file)
+ZWARN             int64                                   Redshift warning flag (from the redrock file)
+SPECTYPE          char[6]                                 Spectroscopic type (from the redrock file)
+DELTACHI2         float64                                 Chi2 difference between the first- and second best redshifts (from the redrock file)
+TARGET_RA         float64 deg                             Right ascension (from the redrock file)
+TARGET_DEC        float64 deg                             Declination (from the redrock file)
+OBJTYPE           char[3]                                 TGT, SKY, BAD, empty (from the redrock file)
+OII_FLUX          float32 1e-17 erg/s/cm^2                Fitted flux for the OII doublet
```
Comment (weaverba137) [inline doc/DESI_SPECTRO_REDUX/SPECPROD/tiles/GROUPTYPE/TILEID/GROUPID/emline-SPECTROGRAPH-TILEID-GROUPID.rst:81]: `10**-17 erg/(s cm2 Angstrom)` is the standard unit.  Only one `/` is allowed.  For the inverse variance: ``10**+34 (s2 cm4 Angstrom2) / erg2``

File: doc/DESI_SPECTRO_REDUX/SPECPROD/tiles/GROUPTYPE/TILEID/GROUPID/emline-SPECTROGRAPH-TILEID-GROUPID.rst [hunk 2]
```diff
@@ -53,113 +47,127 @@ Required Header Keywords
...
+OII_FLUX_IVAR     float32 1e34 / erg^2 * s^2 * cm^4 * A^2 Inverse variance of the fitted flux for the OII doublet
```
Comment (weaverba137) [inline doc/DESI_SPECTRO_REDUX/SPECPROD/tiles/GROUPTYPE/TILEID/GROUPID/emline-SPECTROGRAPH-TILEID-GROUPID.rst:82]: `A` is the unit for amperes.  `Angstrom` is the unit that you're looking for.

File: doc/DESI_SPECTRO_REDUX/SPECPROD/tiles/GROUPTYPE/TILEID/GROUPID/emline-SPECTROGRAPH-TILEID-GROUPID.rst [hunk 3]
```diff
@@ -53,113 +47,127 @@ Required Header Keywords
...
+HALPHA_CHI2       float32                                 Same as OII_CHI2 but for the HALPHA line
+HALPHA_NDOF       int32                                   Same as OII_NDOF but for the HALPHA line
+================= ======= =============================== ===================
 
 
 Notes and Examples
 ==================
 
-*Add notes and examples here.  You can also create links to example files.*
+* The fit is done with the desispec.scripts.emline script.
```
Comment (weaverba137) [inline doc/DESI_SPECTRO_REDUX/SPECPROD/tiles/GROUPTYPE/TILEID/GROUPID/emline-SPECTROGRAPH-TILEID-GROUPID.rst:159]: You don't need a blank line between list items, unless there is a sub-list.

**Review by weaverba137 — APPROVED** [review]
Units need to be fixed up, but otherwise this looks fine.

**Comment by weaverba137** [comment]
@araichoor, I can patch up these units issues in order to get this merged quickly.  However, I noticed a significant discrepancy, even after getting the units into a standard form:
```
OII_FLUX          float32 1e-17 erg/s/cm^2                Fitted flux for the OII doublet
OII_FLUX_IVAR     float32 1e34 / erg^2 * s^2 * cm^4 * A^2 Inverse variance of the fitted flux for the OII doublet
```
The inverse variance had inconsistent units relative to `OII_FLUX`. There are several columns where this is the case.

**Comment by moustakas** [comment]
Also, in the explanation / documentation column: `OII` --> `[OII]`

**Comment by weaverba137** [comment]
@araichoor, thank you this looks fine.  I'm going to go ahead and merge.

**Pull request merged**